### PR TITLE
Standardize MOD30 Readme

### PR DESCRIPTION
**Pull request by paladique**

### CODE REVIEW

**Comment by paladique** [comment]
@FBoucher This is all good to go 👍 Thanks!

File: mod30/README.md [hunk 1]
```diff
@@ -58,15 +77,16 @@ code.
 |[4 - Social Media Integration with Logic Apps ](demos.md#demo-4-social-media-integration-with-logic-apps)|[Link](https://globaleventcdn.blob.core.windows.net/assets/mod/mod30/MOD30_Demo4.mp4)
 |[5 - Automatic Image Captioning with Cognitive Services](demos.md#demo-4-automatic-image-captioning-with-logic-apps)|[Link](https://globaleventcdn.blob.core.windows.net/assets/mod/mod30/MOD30_Demo5.mp4)
 
-## Become a Presenter
+## Become a Trained Presenter
 
-To become a *Trained Presenter*, contact [scalablecontent@microsoft.com](mailto:scalablecontent@microsoft.com). In your email please include:
+To become a trained presenter, contact [learningpathmanager@microsoft.com](mailto:learningpathmanager@microsoft.com). In your email please include:
```
Comment (JeremyLikness) [inline mod30/README.md]: Which email is correct? Did you sync on latest? @lbugnion made some updates today so be sure to sync those first.

File: mod30/README.md [hunk 2]
```diff
@@ -58,15 +77,16 @@ code.
...
 
 - Complete name:
-- The code of this presentation: MOD30
-- Link (ex: unlisted YouTube video) to a video of you presenting (~10 minutes).
+- The code of this presentation: apps10
```
Comment (JeremyLikness) [inline mod30/README.md]: The session code is MOD30, not apps10

**Comment by JeremyLikness** [comment]
@FBoucher I think we need to sync back to @lbugnion changes before merging this.

File: mod30/README.md (hunk 2 above)
Comment (paladique) [reply mod30/README.md]: Will fix, missed when copied from apps10 example

File: mod30/README.md (hunk 1 above)
Comment (paladique) [reply mod30/README.md]: This was updated to be standardized, [based on the apps10 example](https://github.com/microsoft/ignite-learning-paths-training-apps/tree/master/apps10#become-a-trained-presenter). Assumed this email might be correct one. @FBoucher which email should we be using?

**Comment by paladique** [comment]
@JeremyLikness based on @lbugnion's [last commit here](https://github.com/microsoft/ignite-learning-paths-training-mod/commit/e9be3cc9e047acbfd7eb9bee8bf89fbc182a638d), there were no changes to the readme. I have only made changes to the readme, so it's current.

File: mod30/README.md (hunk 1 above)
Comment (paladique) [reply mod30/README.md]: I'm going to use the old one for now till this is clarified

**Comment by paladique** [comment]
Closing this so easier to understand commit history from master.

**Pull request closed without merging**